### PR TITLE
Make the README point to the RTD docs

### PR DESCRIPTION
**Pull request by leotrs**
Title. Having the full installation instructions in the README is now unnecessary as they now live in RTD.

I also changed the docs badge from @eulertour's to the official RTD badge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,272 +1,65 @@
...
+        square.rotate(-3 * TAU / 8)
+        circle.set_fill(PINK, opacity=0.5)
+
+        self.play(ShowCreation(square))
+        self.play(Transform(square, circle))
+        self.play(FadeOut(square))
 ```
 
-2. Run the following command:
+Save this code in a file called `basic.py`.  Now open your terminal in the
```
Comment (huguesdevimeux) [inline README.md:48]: There is already SquareToCircle in `example_scenes/basic.py`, no ? DO i miss something ?

File: README.md [hunk 2]
```diff
@@ -1,272 +1,65 @@
...
+folder where you saved the file and execute
 ```sh
 manim example_scenes/basic.py SquareToCircle -pl
```
Comment (behackl) [inline README.md:51]: The path `example_scenes/basic.py` might not be correct; the tutorial now just says to name the file `basic.py`.

**Comment by leotrs** [comment]
To answer the two comments left just now @huguesdevimeux @behackl. My intention was to not mention example_scenes/basic.py at all in the README, I think it's unnecessary and can even be confusing. I think that providing a code sample and asking the user to save it in a file themselves makes more sense as "example usage".

But I may be wrong, I don't feel strongly about this.

So, either we change the recommended name of the file from `basic.py` to `example.py` or something like that, OR we remove the example code block and just ask them to run `example_scenes/basic.py`.

Note for the latter to work we will have to make sure that example_scenes are part of the pypi package cc @naveen521kk

**Comment by behackl** [comment]
> So, either we change the recommended name of the file from `basic.py` to `example.py` or something like that, OR we remove the example code block and just ask them to run `example_scenes/basic.py`.

Maybe both? I like that you provide a small example that can be copy/pasted (but certainly :+1: for naming it `example.py`). On the other hand, mentioning the `example_scenes` folder also would not hurt. :shrug:

**Comment by naveen521kk** [comment]
> Note for the latter to work we will have to make sure that example_scenes are part of the pypi package

Do any of know package include example in pip packages?
Also, it will located somewhere hidden in `site-packages` folder.

**Comment by leotrs** [comment]
As for including examples in the package: pandas and scikit-learn do this. You can import example data sets as `from sklearn.datasets import xxx` or something like that.

But that would mean to move `example_scenes/` inside `manim/`, which I don't think is a good idea. The purpose of `example_scenes/` is to showcase some scene code, not to export something to the user they can then build upon.

**Comment by naveen521kk** [comment]
So let users download the example scenes or copy paster from docs.

**Comment by leotrs** [comment]
@huguesdevimeux @behackl done!

**Review by behackl — APPROVED** [review]
LGTM, it really makes the README much more readable. Good work!

**Review by huguesdevimeux — APPROVED** [review]

**Pull request merged**